packages/components/atoms/f-button/jest.config.js

```diff
@@ -37,8 +37,4 @@ module.exports = {
     ],
 
     testURL: 'http://localhost/',
-
```

Comment:
just need to remove the `cd ../../../../ && yarn allure:clean` from the package.json test script for these components too 👍

Reply:
done!